src/components/AboutContainer/AboutContentContainer/AboutContentPages/GetInvolved.js

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import Button from '@material-ui/core/Button';
+import useStyles from '../AboutContentContainer.styles';
+import { withStyles } from '@material-ui/core/styles';
+
+const BlueOutlineButton = withStyles({
+  root: {
+    background: 'transparent',
+    borderRadius: '4px',
+    border: '1px solid #016EE3',
+    height: '45px',
+    padding: '10px 21px',
+    marginTop: 'auto',
+    color: '#026FE4',
+    boxShadow: 'none',
+    '&:hover': {
+      background: '#550CCC',
+      border: 'none',
+      color: 'white',
+    },
+    '@media (max-width: 799px)': {
+      alignSelf: 'center',
+      marginTop: 'auto',
+      marginBottom: '20px',
+    },
+  },
+  label: {
+    fontFamily: 'Open Sans, sans-serif',
+    fontWeight: 600,
+    size: '18px',
+  },
+})(Button);
+
+const GetInvolved = () => {
+  const { title } = useStyles();
+  return (
+    <>
+      <Typography variant="h2" className={title}>
+        We are proudly volunteer-driven
+      </Typography>
+      <Grid
+        container
+        direction="row"
+        justify="flex-start"
+        alignItems="flex-start"
+        spacing={7}
```

Comment:
let's make this es6

`{...{container: true,
        direction: "row",
        justify: "flex-start",
        alignItems: "flex-start",
        spacing: 7}}`